rasa/core/channels/slack.py

```diff
@@ -375,8 +377,8 @@ async def process_message(
                 metadata=metadata,
             )
 
-            await on_new_message(user_msg)
-        except Exception as e:
+            asyncio.get_event_loop().create_task(on_new_message(user_msg))
+        except (Exception, asyncio.CancelledError) as e:
```

Comment:
> asyncio.CancelledError

Can this error even happen as the `try .. except` block will already be left when this happens?